src/cpp/statistics/types/monitorservice_typesCdrAux.hpp

```diff
@@ -37,24 +37,24 @@ constexpr uint32_t eprosima_fastdds_statistics_DeadlineMissedStatus_s_max_cdr_ty
 constexpr uint32_t eprosima_fastdds_statistics_DeadlineMissedStatus_s_max_key_cdr_typesize {0UL};
 
 
+
 constexpr uint32_t eprosima_fastdds_statistics_QosPolicyCount_s_max_cdr_typesize {12UL};
 constexpr uint32_t eprosima_fastdds_statistics_QosPolicyCount_s_max_key_cdr_typesize {0UL};
 
 
-
-constexpr uint32_t eprosima_fastdds_statistics_MonitorServiceStatusData_max_cdr_typesize {565252UL};
+constexpr uint32_t eprosima_fastdds_statistics_MonitorServiceStatusData_max_cdr_typesize {72UL};
```

Comment:
What a big decrease in the number!!

Reply:
Looking at the IDL file, the MonitorServiceStatusData is a structure which has a union member. This union has a member which is a sequence of a structure (Connection) that contains two more sequences. Consequently, the size reduction would be caused by the change in behavior in Fast DDS-Gen of allocating only 50 elements in unbounded collections, instead of 100.